src/plugins/health.ts

```diff
@@ -0,0 +1,232 @@
+import {Plugin, Cordova} from './plugin';
+
+/**
+ * @param: startDate: {type: Date}, start date from which to get data
+ * @param: endDate: {type: Date}, end data to which to get the data
+ * @param: dataType: {type: String}, the data type to be queried (see below for supported data types)
```

Comment:
Documentation for interfaces should look like this:

```
export interface QueryOptions { 
   ...
   /**
   * The data type to be queried
   */
   dataType: string;
   ...
}
```

Each property needs to have its definition right above it, there is no need to have the type there as it's defined in the typings and the doc generator is able to read that.